### ISSUE TITLE
Single person template - remove text clipping on phone number, email

### ISSUE THREAD

**Issue by cjg89** [UCF/Colleges-Theme#68]
**Describe the bug**

https://github.com/UCF/Colleges-Theme/blob/d480c015d819559c1dd70049505fec6561b49ace/src/scss/_person.scss#L36-L43

`overflow: hidden;`, `text-overflow: ellipsis;`, and `white-space: nowrap;` should be removed from the snippet linked above to ensure that users zoomed up to 200% will not have phone/email contents clipped that users at a standard zoom level would be able to see.

**Steps to Reproduce**

n/a

**Expected Behavior**

Text should never be clipped/truncated more when users zoom their browser up to 200%.

**Possible Fix**

Remove the rules noted above.

**Your environment:**

n/a

**Additional context**

https://www.w3.org/WAI/WCAG21/Understanding/resize-text
